Sources/Plot/API/HTMLElements.swift

```diff
@@ -471,3 +471,11 @@ public extension Node where Context: HTMLScriptableContext {
         .element(named: "script", nodes: nodes)
     }
 }
+
+public extension Node where Context == HTML.ScriptContext {
+    /// Add a `async` attribute within the current context.
+    static var async : Node {.attribute(named: "async", value: nil, ignoreIfValueIsEmpty: false)}
```

Comment:
Please follow the same code style as the rest of the project. If you look how other APIs within the same file is defined, you'll see that their style differs quite a lot from the one you've used, which'll make the code unnecessarily inconsistent.

Reply:
Also, Plot doesn't use computed properties as part of its API, further making these additions feel quite inconsistent with the rest of the code base. Would be great if you could define them similar to how the `.controls()`, `.autocomplete()`, and `.required()` attributes are defined, just to name a few examples.